spot-client/src/common/remote-control/remoteControlClient.js

```diff
@@ -62,8 +62,7 @@ export class RemoteControlClient extends BaseRemoteControlService {
      * @returns {Promise} Resolves if the command has been acknowledged.
      */
     adjustVolume(direction) {
-        return this.xmppConnection.sendCommand(
-            this._getSpotId(), COMMANDS.ADJUST_VOLUME, { direction });
+        return this._sendCommand(COMMANDS.ADJUST_VOLUME, { direction });
```

Comment:
This change could go in on its own? Just trying to figure out stuff we can get in earlier to make the diff clearer.

Reply:
yes